java/pending_code_changes.txt

```diff
@@ -1 +1,11 @@
+Pending metadata changes:
+* Metadata fixes for BY, KR, NA
+
+Pending code changes:
```

Comment:
Let's remove "Pending", from above too.
In this file we make it as easy as possible to merge automatically during the release.
The file is already called "pending", here we make it just "Code changes" and "Metadata changes". That part currently does not merge automatically into the release notes (b/31174165).